about.js

```diff
@@ -0,0 +1,61 @@
+const hambuger = document.getElementById('hambuger');
+const body = document.getElementsByTagName('body')[0];
+hambuger.addEventListener('click', () => {
+  const welcom = document.getElementsByClassName('welcome')[0];
+  welcom.style = 'margin-top:70px;';
+  hambuger.style = 'display:none';
+  const div = document.createElement('div');
+  div.style = `padding: 15px 20px;
+               background-color:rgba(104, 103, 103, 0.8);
+               color: #fff;
+               position:fixed;
+               right:0;
+               top:0;
+               left:0;
+               bottom:0;
+               height:100vh;
+               z-index:2;`;
+  div.innerHTML = `
+    <img id="cross" src="./images/icons/cross.svg" style="width:40px;" alt="Cross Icon">
+    <div style="display:flex;flex-direction:column;align-items:center;margin-top:30px;gap:10px;">
+      <a href="./index.html" style="text-decoration:none;color:#fff;font-size:25px;">Home</a>
+      <a href="#" id="programLink" style="text-decoration:none;color:#fff;font-size:25px;">Program</a>
+      <a href="#" style="text-decoration:none;color:#fff;font-size:25px;">Register</a>
+      <a href="#" id="partnerLink" style="text-decoration:none;color:#fff;font-size:25px;">Sponsor</a>
+      <a href="#" style="text-decoration:none;color:#fff;font-size:25px;">News</a>
+      <a href="#" style="text-decoration:none;color:#fff;font-size:25px;">Goto VarCamp</a>
+    </div>
+  `;
+  body.appendChild(div);
+
+  const about = document.getElementById('about');
+  const logo = document.getElementById('logo');
+  const concert = document.getElementById('concert');
+  const partner = document.getElementById('about-partner');
+  const footer = document.getElementById('about-footer');
+  about.style = 'filter:blur(3px)';
+  logo.style = 'filter:blur(3px)';
+  concert.style = 'filter:blur(3px)';
+  partner.style = 'filter:blur(3px)';
+  footer.style = 'filter:blur(3px)';
+
+  function removeMobileMenu() {
+    welcom.style = `
+      @media screen and (max-width: 768px) {
+        margin-top:0;
+      }
+    `;
+    hambuger.style = 'display:block;';
+    body.removeChild(div);
+    about.style = 'filter:blur(0)';
+    logo.style = 'filter:blur(0)';
+    concert.style = 'filter:blur(0)';
+    partner.style = 'filter:blur(0)';
+    footer.style = 'filter:blur(0)';
+  }
+
+  const cross = document.getElementById('cross');
+  cross.addEventListener('click', () => {
+    removeMobileMenu();
+  });
+});
```

Comment:
- [ ] Nice implementation :+1: but your code is not DRY. DRY, which stands for 'don't repeat yourself,' is a principle of software development that aims at reducing the repetition of patterns and code duplication in favour of abstractions and avoiding redundancy. :slightly_smiling_face: 
- [ ] Check my suggestion below :point_down: 

```suggestion
const hamburger = document.getElementById('hamburger');
const body = document.getElementsByTagName('body')[0];

const createMobileMenu = () => {
  const welcome = document.querySelector('.welcome');
  welcome.style.marginTop = '70px';

  hamburger.style.display = 'none';

  const div = document.createElement('div');
  div.style.cssText = `padding: 15px 20px;
    background-color: rgba(104, 103, 103, 0.8);
    color: #fff;
    position: fixed;
    right: 0;
    top: 0;
    left: 0;
    bottom: 0;
    height: 100vh;
    z-index: 2;`;

  div.innerHTML = `
    <img id="cross" src="./images/icons/cross.svg" style="width:40px;" alt="Cross Icon">
    <div style="display:flex;flex-direction:column;align-items:center;margin-top:30px;gap:10px;">
      <a href="./index.html" style="text-decoration:none;color:#fff;font-size:25px;">Home</a>
      <a href="#" id="programLink" style="text-decoration:none;color:#fff;font-size:25px;">Program</a>
      <a href="#" style="text-decoration:none;color:#fff;font-size:25px;">Register</a>
      <a href="#" id="partnerLink" style="text-decoration:none;color:#fff;font-size:25px;">Sponsor</a>
      <a href="#" style="text-decoration:none;color:#fff;font-size:25px;">News</a>
      <a href="#" style="text-decoration:none;color:#fff;font-size:25px;">Goto VarCamp</a>
    </div>
  `;

  body.appendChild(div);

  const blurElements = document.querySelectorAll('#about, #logo, #concert, #about-partner, #about-footer');

  blurElements.forEach(element => {
    element.style.filter = 'blur(3px)';
  });

  const removeMobileMenu = () => {
    welcome.style.marginTop = `
      @media screen and (max-width: 768px) {
        margin-top:0;
      }
    `;

    hamburger.style.display = 'block';

    body.removeChild(div);

    blurElements.forEach(element => {
      element.style.filter = 'blur(0)';
    });
  };

  const cross = document.getElementById('cross');
  cross.addEventListener('click', removeMobileMenu);
};

hamburger.addEventListener('click', createMobileMenu);
```

**This version is drier because it:**

- Uses `querySelector` instead of `getElementsByClassName` and `getElementById`.
- Uses `querySelectorAll` and `forEach` to loop over elements that need to be blurred and unblurred.
- Moves the `removeMobileMenu` function inside the `createMobileMenu` function, making it a closure that has access to the variables in `createMobileMenu`.

Reply:
Fixed.